### DISCUSSION TITLE
Welcome to coffee-tasks Discussions!

### DISCUSSION THREAD

**Discussion by SauveJeanLuc** [SauveJeanLuc/coffee-tasks#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aleksanderwalczuk** [comment]
Hi 👋 I'm a Frontend Developer based in Poland. I want to learn react and I think this is a great place for that. On daily basis I work in Vue.js and Node. Nice to meet you ✋
